scripts/test-configs/alice.toml

```diff
@@ -0,0 +1,8 @@
+# Alice's test Config
+
+usedurablestore = true
```

Comment:
If we removed this (and maybe let the script handle it) the test config files only contain actor specific info.

Reply:
I considered that, but in the end I liked a more production-y setup which was more explicit and flexible. Definitely an option, though.